src/utils/config_filters.hpp

```diff
@@ -69,4 +72,11 @@ bool string_matches_if_present(
 
 bool bool_or_empty(const config& filter, const config& cfg, const std::string& attribute);
 
+/**
+ * vector used for verify what attribute checked can be present in special/abilities.
+ */
+const std::set<std::string> abilities_tags{"tailwind", "hides", "skirmisher", "teleport", "leadership", "illuminates", "heals", "regenerate"};
```

Comment:
The whole PR makes the logic feel more convoluted, more places where specific abilities are given odd edge cases.

Reply:
i will perhaps close this PR then because between weapon andabilities not hard enginered, it si became impossible to sais if 'value' can be codedor not. I will await to see end checking.